pyidr/study_parser.py

```diff
@@ -18,78 +18,85 @@
 
 class Key(object):
 
-    def __init__(self, pattern, scope, optional=False):
-        self.pattern = pattern
+    def __init__(self, name, scope, optional=False, multiple=False):
+        self.name = name
         self.scope = scope
         self.optional = optional
+        self.multiple = multiple
 
 
 KEYS = (
     # OPTIONAL_KEYS["Study"]
     Key(r'Comment\[IDR Study Accession\]', 'Study'),
     Key('Study Title', 'Study'),
     Key('Study Description', 'Study'),
-    Key('Study Type', 'Study'),
-    Key('Study Type Term Source REF', 'Study', optional=True),
-    Key('Study Type Term Accession', 'Study', optional=True),
-    Key('Study Publication Title', 'Study'),
-    Key('Study Author List', 'Study'),
-    Key('Study Organism', 'Study', optional=True),
-    Key('Study Organism Term Source REF', 'Study', optional=True),
-    Key('Study Organism Term Accession', 'Study', optional=True),
+    Key('Study Type', 'Study', multiple=True),
```

Comment:
Looking at it now, I could see these Keys being either parsed out of a schema representation somewhere or available in a non-util library.

Reply:
Absolutely, as I was extending the `Key` class, I very much felt that I was rewriting (poorly) `jsonschema`. If we agree with the value, I can definitely look into turning this into a re-usable schema representation (and that can in turn be used more generically).